### PR TITLE
Multiblob support (on twojs experimental branch)

### PR DESCRIPTION
**Pull request by JoeMattie**
I kludged in multi-blob support.  It needs a bit of work, but it works if you would like to pull this branch and check it out.

You spawn with two blobs at the moment to test it out.  I'll refactor to handle everything a bit more sanely shortly.  At the moment, I haven't written the remerge logic, so the blobs just bounce off of each other.

I'm finishing up a fairly pressing project this week, but starting next week I'll have a lot more time to devote to cleaning this up and polishing it.

### CODE REVIEW

**Comment by Buskcoin** [comment]
I wan't to try it but I can't get it to work :/ .

**Comment by JoeMattie** [comment]
what errors are you getting?

**Comment by Buskcoin** [comment]
* When I run `npm install`:

/agar/server/server.js: line 73, col 16, You might be leaking a variable (xMax) here. (W120)
/agar/server/server.js: line 74, col 16, You might be leaking a variable (yMax) here. (W120)

2 errors
[07:04:31] Finished 'build-server' after 180 ms
[07:04:31] Finished 'move-client' after 302 ms
[07:04:31] Starting 'build-client'...
/agar/client/js/app.js: line 45, col 20, ['webgl'] is better written in dot notation. (W069)
/agar/client/js/app.js: line 60, col 2, Missing semicolon. (W033)
/agar/client/js/app.js: line 88, col 22, Expected an assignment or function call and instead saw an expression. (W030)
/agar/client/js/app.js: line 301, col 2, Unnecessary semicolon. (W032)
/agar/client/js/app.js: line 427, col 23, 'i' is already defined. (W004)
/agar/client/js/app.js: line 551, col 23, 'a' is already defined. (W004)
/agar/client/js/app.js: line 560, col 23, 'a' is already defined. (W004)
/agar/client/js/app.js: line 568, col 23, 'a' is already defined. (W004)
/agar/client/js/app.js: line 604, col 15, 'a' is already defined. (W004)

9 errors

* When i run `node server/server.js`:

module.js:340
    throw err;
          ^
Error: Cannot find module 'MD5'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (home/agar/server/server.js:7:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)

**Comment by JoeMattie** [comment]
That's only one error, the rest are warnings...

Fix it by doing:

    npm install -s MD5

**Comment by JuhQ** [comment]
@JoeMattie please add md5 package to package.json.

File: server/server.js [hunk 1]
```diff
@@ -25,7 +27,7 @@ function addFood(toAdd) {
     while(toAdd--){
         food.push({
             // make ids unique
-            id: ((new Date()).getTime() + '' + (new Date()).getMilliseconds() + '' + food.length) >>> 0,
+            id: md5((new Date()).getTime() + '' + (new Date()).getMilliseconds() + '' + food.length),
```
Comment (Amoki) [inline server/server.js]: May could we use uuid that are designed for that ?
https://www.npmjs.com/package/uuid

File: server/server.js [hunk 2]
```diff
@@ -66,35 +68,78 @@ function massToRadius(mass){
...
 
-    var deltaY = player.speed * Math.sin(deg)/ slowDown;
-    var deltaX = player.speed * Math.cos(deg)/ slowDown;
+    var xMin = xMax = playerObjects[0].x;
+    var yMin = yMax = playerObjects[0].y;
 
-    if (dist < (50 + player.mass)) {
-        deltaY *= dist / (50 + player.mass);
-        deltaX *= dist / (50 + player.mass);
+    for(var i=1; i < playerObjects.length; ++i)
```
Comment (Amoki) [inline server/server.js]: playerObjects.forEach() to keep the same syntax for each loop ? (pun not intended)
If i'm boring, tell me now :p

Comment (JoeMattie) [inline server/server.js]: As I understand it, the for loop is much more performant than foreach

**Pull request closed without merging**